Makefile

```diff
@@ -37,7 +37,7 @@ tfgen:: install_plugins
 provider:: tfgen install_plugins # build the provider binary
 	(cd provider && go build -a -o $(WORKING_DIR)/bin/${PROVIDER} -ldflags "-X ${PROJECT}/${VERSION_PATH}=${VERSION}" ${PROJECT}/${PROVIDER_PATH}/cmd/${PROVIDER})
 
-build_sdks:: install_plugins provider build_nodejs build_python build_go build_dotnet build_java  # build all the sdks
+build_sdks:: install_plugins provider build_nodejs build_python build_go build_dotnet # build_java  # build all the sdks
```

Comment:
Why do you disable the java build? Especially after the `upgrade-provider` command added the `.pulumi-java-gen.version` file. IIRC you mentioned in the past, that there is no location to publish the java build.
However, the repo still contains the Java SDK. We can either get rid of the java SDK all together or we continue building it. My preference is the later as I see value for users.